src/slave/containerizer/mesos/isolators/namespaces/ipc.cpp

```diff
@@ -48,6 +50,23 @@ namespace mesos {
 namespace internal {
 namespace slave {
 
+// Small C helper which looks up the login of a user in passwd
+// and returns its UID. Handy to provide right perms on mountpoints
+// We should use the reentrant version as it's unsure this can be called
```

Comment:
which re-entrant version?

Reply:
yes sorry, i meant `getpwnam` is not reentrrant